website/app/sources/local/search.js

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
Is this a copy of the existing file or has the other one been removed? Does this need to be duplicated?

Reply:
It needs to be wrapped as shown below because it uses jQuery - e.g. `if (typeof module === 'object') {window.module = module; module = undefined;}` at the start.

This wouldn't affect the normal we script if we put it in that, so could do that?

Reply:
Resolved in 05ff6aa